### PR TITLE
add autoconfig & autodiscover

### PR DESCRIPTION
**Pull request by docjyJ**

### CODE REVIEW

**Comment by docjyJ** [comment]
https://stalw.art/docs/server/autoconfig/

File: start.sh [hunk 1]
```diff
@@ -82,6 +82,38 @@ https://mail.{\$NC_DOMAIN}:443 {
         }
     }
 }
+https://autoconfig.{\$NC_DOMAIN}.fr:443 {
+    # import GEOFILTER
```
Comment (szaimen) [inline start.sh:86]: I would probably remove the geofilter

Comment (docjyJ) [reply start.sh]: done

**Comment by docjyJ** [comment]
ready

**Pull request merged**